src/test/java/org/openmrs/contrib/qaframework/page/EditVitalsPage.java

```diff
@@ -0,0 +1,110 @@
+package org.openmrs.contrib.qaframework.page;
```

Comment:
add a header license to EditVitalsPage 
/**
 * This Source Code Form is subject to the terms of the Mozilla Public License,
 * v. 2.0. If a copy of the MPL was not distributed with this file, You can
 * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
 * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
 *
 * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
 * graphic logo is a trademark of OpenMRS Inc.
 */
package org.openmrs.contrib.qaframework.page;

import org.openmrs.contrib.qaframework.helper.Page;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

public class EditVitalsPage extends Page {

}

Reply:
@kdaud I have attended to the review